packages/synapse-react-client/.storybook/main.mts

```diff
@@ -18,7 +18,7 @@ const config: StorybookConfig = {
   framework: {
     name: '@storybook/react-vite',
     options: {
-      strictMode: false,
+      strictMode: true,
```

Comment:
StrictMode seems to be working, so enable it and hopefully we can catch more bugs like this